test/support/utils.js

```diff
@@ -88,7 +97,7 @@ function rawrequest(server) {
 function shouldHitHandle(num) {
   var header = 'x-fn-' + String(num)
   return function (res) {
-    assert.equal(res.headers[header], 'hit')
+    assert.equal(res.headers[header], 'hit', 'didn\'t hit ' + num)
```

Comment:
assertion messages are support to describe the _wanted_ outcome, rather than the error